### DISCUSSION TITLE
output (stylish formatter): print relative paths

### DISCUSSION THREAD

**Discussion by mustafa0x** [eslint/eslint#19212] [Q&A]
Absolute paths add a lot of noise to the output.

- Continuation of: https://github.com/eslint/eslint/issues/13376
- see https://eslint.org/docs/latest/use/formatters/#stylish
- possibly related: https://github.com/a-tarasyuk/eslint-formatter-autolinkable-stylish

**Comment by nzakas** [comment]
You can always make your own formatter and have it output exactly what you want. We're not creating any more formatters in the core.

**Reply by mustafa0x** [reply]
Thanks. Perhaps this can be an option?

If not, any advice on making a formatter to print paths relative to the root folder?

**Reply by nzakas** [reply]
Formatters don't have options, and as I mentioned, we're not adding any further formatters to the core.

I'd say take a look at the existing stylish formatter then use `path.relative()` to output the file paths.
